pages/lazer/subscribe-price-updates.mdx

```diff
@@ -35,64 +37,64 @@ client.send({
-- `channel` allows to configure the update rate: updates in the **real_time** channel are sent as frequently as possible, while **fixed_rate@200ms** and **fixed_rate@50ms** channels are updated at fixed rates.
+- `subscriptionId` is an arbitrary numeric identifier for a subscription. It will be returned back in response by the server. It does not affect the signed payload.
+- `priceFeedIds` is the list of price feeds to receive updates for. Data for all price feeds will be present in the signed price updates generated. Refer to the [Price Feed IDs list](./price-feed-ids.mdx) for the supported price feeds.
+- `properties` is the list of properties to retrieve, such as **price**, **bestBidPrice**, **bestAskPrice**, etc.
+- `chains` is the list of chains to receive a signed payload for, such as **evm**, **solana**, etc.
+- `channel` determines the update rate: updates in the **real_time** channel are sent as frequently as possible, while **fixed_rate@200ms** and **fixed_rate@50ms** channels are updated at fixed rates.
 
-There are also a few other parameters one may use. Refer to the [API documentation](https://pyth-lazer.dourolabs.app/docs) for more details.
+There are also a few other configuration parameters -- see the [API documentation](https://pyth-lazer.dourolabs.app/docs) for more details.
+
+Determine the most suitable values for your application -- they will be used in the next step.
 
 ### 3. Subscribe to the price updates
 
-To subscribe to the price updates, one needs to send the request to the websocket server. The server will respond with a signed price update.
+To subscribe to the price updates, send a request to the websocket server. The server will respond with a signed price update.
 
-1. Pyth Lazer provides a [SDK](https://github.com/pyth-network/pyth-crosschain/tree/main/lazer/sdk/js) to seamlessly integrate the websocket API into your application.
-   It can be installed using the following command:
+1. Pyth Lazer provides an [SDK](https://github.com/pyth-network/pyth-crosschain/tree/main/lazer/sdk/js) to seamlessly integrate the websocket API into your application.
+   Install it using the following command:
 
-```bash
+```bash copy
 npm install --save @pythnetwork/pyth-lazer-sdk
 ```
 
-2. Then create a [`PythLazerClient`](https://github.com/pyth-network/pyth-crosschain/blob/main/lazer/sdk/js/src/client.ts#L32) object using the URL and the access token requested from the Pyth team in the first step.
+2. Then create a [`PythLazerClient`](https://github.com/pyth-network/pyth-crosschain/blob/main/lazer/sdk/js/src/client.ts#L32) object using the URL and the access token requested from the Pyth team in the first step:
 
-```js
+```js copy
 import { PythLazerClient } from "@pythnetwork/pyth-lazer-sdk";
 
-const client = new PythLazerClient(
-  "wss://pyth-lazer.dourolabs.app/v1/stream",
-  "ctoken1"
+const client = await PythLazerClient.create(
```

Comment:
it looks like the SDK interface changed since this was written

(by the way the example is wrong now as well)